public/app/features/explore/Explore.tsx

```diff
@@ -73,18 +73,6 @@ const getStyles = stylesFactory(() => {
     button: css`
       margin: 1em 4px 0 0;
     `,
-    // Utility class for iframe parents so that we can show iframe content with reasonable height instead of squished
-    // or some random explicit height.
-    fullHeight: css`
```

Comment:
Was used before when we used iframe for the trace ui

Reply:
👍